.golangci.yml

```diff
@@ -57,6 +57,10 @@ linters-settings:
     simple: true
     range-loops: true
     for-loops: false
+  gosec:
+    excludes:
+      - G108
+      - G114
```

Comment:
Please don't disable gosec,  consider using  // nolint tag

Reply:
Unfortunately the // nolint tag does not disable the check

Reply:
Have you looked into #nosec tag?

Reply:
Not yet, I'll check

Reply:
Adding the #nosec tag to the pprof package import disables the warning. Thanks!